src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1060,4 +1060,10 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Hyperlink_Owner" xml:space="preserve">
     <value>Owner</value>
   </data>
+  <data name="ToolTip_UseDotnetCommand" xml:space="preserve">
+    <value>To know this package transitive origin path use command</value>
```

Comment:
I don't think this will be enough information for people unfamiliar with command lines in general, and they might search Visual Studio's "features" (ctrl q) for it.

Next, SDK style proejcts have can use VS's Solution Explorer (although they need to enable "search external files").

For non-SDK style projects, customers we don't currently have anything better than dotnet nuget why, but in many of those cases, they can't use `dotnet nuget why my.csproj`. They'll have to restore in VS or with msbuild.exe, and then they'll need to pass the assets file as the "project" argument.  Hence, I don't think that just mentioning dotnet nuget why is enough.  We should at least point them to https://aka.ms/dotnet/nuget/why so they might see the docs that explain the restore + assets file steps.

But I think it would be better if we had an aka.ms link for understanding transitive dependencies. For now we can point it to the nuget 2.0 blog post, and if we improve our learn docs, then we can redirect it to those future docs, so customers using SDK style projects aren't effectively told to use dotnet nuget why on the CLI for something they can do without leaving VS.

Reply:
Since it's a tooltip we cannot add links to it or anything clickable 😢.

Reply:
The URL is short enough that hopefully customers can memorise and type it out. I think it's better than telling someone who doesn't know what dotnet nuget why is, or unfamiliar with the command line in general, to run the command without any additional context.

Reply:
I added the link to the tooltip text